src/python/CRABClient/Commands/preparelocal.py

```diff
@@ -65,10 +68,19 @@ def getInputFiles(self):
         crabDBInfo, _, _ = server.get(api='task', data={'subresource': 'search', 'workflow': taskname})
         status = getColumn(crabDBInfo, 'tm_task_status')
         self.destination = getColumn(crabDBInfo, 'tm_asyncdest')
+        username = getColumn(crabDBInfo, 'tm_username')
+        sandboxName = getColumn(crabDBInfo, 'tm_user_sandbox')
 
         inputsFilename = os.path.join(os.getcwd(), 'InputFiles.tar.gz')
+        sandboxFilename = os.path.join(os.getcwd(), 'sandbox.tar.gz')
         if status == 'UPLOADED':
```

Comment:
do we still need this "if" ? We could always pull files from S3, instead of scheduler.

Reply:
Good catch. 
It should be `if status in ['UPLOADED', 'SUBMITTED']:`.

Reply:
OK